docs/source/json-api/index.rst

```diff
@@ -1234,6 +1234,424 @@ HTTP Response
+        },
+        {
+          "type": "CanReadAs",
+          "party": "Bob",
+        },
+        {
+          "type": "ParticipantAdmin"
+        }
+      ]
+    }
+
+Please refer to :ref:`GrantUserRightsRequest <com.daml.ledger.api.v1.admin.user_management_service.GrantUserRightsRequest>` documentation for information about the meaning of the fields.
+
+HTTP Response
+=============
+
+.. code-block:: json
+
+    {
+      "result": [
+        {
+          "type": "CanActAs",
+          "party": "Carol"
+        },
+        {
+          "type": "CanReadAs",
+          "party": "Alice",
+        },
+        {
+          "type": "CanReadAs",
+          "party": "Bob",
+        },
+        {
+          "type": "ParticipantAdmin"
+        }
+      ],
+      "status": 200
+    }
+
+Returns back the rights which were actually applied as existing rights are ignored.
+
+Revoke User Rights
+*****************
+
+This endpoint is a JSON API proxy for the Ledger API's :ref:`RevokeUserRightsRequest <com.daml.ledger.api.v1.admin.user_management_service.RevokeUserRightsRequest>`. For more information about user management, please refer to :ref:`Provisioning Identifiers <provisioning-ledger-identifiers>` part of the Ledger API documentation.
+
+HTTP Request
+============
+
+- URL: ``/v1/user/rights/revoke``
+- Method: ``POST``
+- Content-Type: ``application/json``
+- Content:
+
+.. code-block:: json
+
+    {
+      "userId": "Carol",
+      "rights": [
+        {
+          "type": "CanActAs",
+          "party": "Carol"
+        },
+        {
+          "type": "CanReadAs",
+          "party": "Alice",
+        },
+        {
+          "type": "CanReadAs",
+          "party": "Bob",
+        },
+        {
+          "type": "ParticipantAdmin"
+        }
+      ]
+    }
+
+Please refer to :ref:`RevokeUserRightsRequest <com.daml.ledger.api.v1.admin.user_management_service.RevokeUserRightsRequest>` documentation for information about the meaning of the fields.
+
+HTTP Response
+=============
+
+.. code-block:: json
+
+    {
+      "result": [
+        {
+          "type": "CanActAs",
+          "party": "Carol"
+        },
+        {
+          "type": "CanReadAs",
+          "party": "Alice",
+        },
+        {
+          "type": "CanReadAs",
+          "party": "Bob",
+        },
+        {
+          "type": "ParticipantAdmin"
+        }
+      ],
+      "status": 200
+    }
+
+Returns back the rights which were actually revoked as non-existing rights are ignored.
+
+List Authenticated User Rights
+******************************
+
+This endpoint is a JSON API proxy for the Ledger API's :ref:`ListUserRightsRequest <com.daml.ledger.api.v1.admin.user_management_service.ListUserRightsRequest>`. For more information about user management, please refer to :ref:`Provisioning Identifiers <provisioning-ledger-identifiers>` part of the Ledger API documentation.
+
+The user id will always be filled out with the user specified via the currently used User Token.
```

Comment:
I'm not entirely sure which one we want here, but if User Token is capitalized I'd also capitalize User ID. Or neither.

Reply:
Let's just not capitalize it, after all we are not in German ._.